### ISSUE TITLE
Add binaries to releases for benchmark purposes

### ISSUE THREAD

**Issue by dalisoft** [zauni/commitlint-rs#1]
Hi @zauni 

Thank you for your project and hard work to make such tool and sharing with community.
Could you please add **binaries** on **GitHub releases** so i could use your tool ([example](https://github.com/dalisoft/commitlint-rs-npm)) and add to [benchmark](https://github.com/dalisoft/commitlint-sh/tree/development/benchmark)?

**Comment by zauni** [comment]
Hi @dalisoft interesting project you have going on there and also cool that you want to benchmark them all.

I will notify you, as soon as I have binaries ready on GitHub releases.

**Comment by dalisoft** [comment]
Thank you, i will wait for binaries

**Comment by dalisoft** [comment]
I locally build with `cargo build --release`, moved binary and ran benchmark. Currently results are these

| Command                                        |   Mean [ms] | Min [ms] | Max [ms] |      Relative |
| :--------------------------------------------- | ----------: | -------: | -------: | ------------: |
| `echo "fix: it works" \| ../validate.sh`       |   6.6 ± 3.4 |      3.5 |     12.4 |   2.21 ± 1.27 |
| `echo "fix: it works" \| ./commitlint-rs`      |   3.3 ± 0.3 |      2.9 |      3.9 |   1.11 ± 0.32 |
| `echo "fix: it works" \| ./commitguard`        |   3.0 ± 0.8 |      2.1 |      4.7 |          1.00 |
| `echo "fix: it works" \| ./commitlint-go lint` |   3.5 ± 0.7 |      2.5 |      4.8 |   1.18 ± 0.39 |
| `echo "fix: it works" \| commitlint`           | 180.0 ± 6.6 |    170.4 |    190.3 | 60.03 ± 16.40 |

**Comment by zauni** [comment]
Yeah sorry, didn't get to do release anything yesterday. But thanks for the insights in your benchmark! So commitguard is not that bad, but probably because it has only 5 rules or so 😄

**Comment by dalisoft** [comment]
@zauni Build or release isn't urgent, i just so wanted to test your tool performance and did it and share benchmark results with you. It is very fast.
